ariadne/app-qt/Glue.hs

```diff
@@ -204,11 +205,14 @@ cardanoEventToUI = \case
   CardanoStatusUpdateEvent CardanoStatusUpdate{..} ->
     Just $ UiCardanoEvent $
       UiCardanoStatusUpdateEvent UiCardanoStatusUpdate
-        { tipHeaderHash = pretty tipHeaderHash
-        , tipSlot = pretty tipEpochOrSlot
-        , currentSlot = pretty currentSlot
-                     <> if isInaccurate then " (inaccurate)" else ""
+        { syncProgress = syncProgress >>= \frac -> return $ toFixed 1 (fromRational $ frac * 100) <> "%"
```

Comment:
Nitpick: is it the same as `toFixed 1 . fromRational . (*100) <$> syncProgress`?

Reply:
Uh-huh, `(<> "%") .`